server/helpers/planner_helper/planner_helper.py

```diff
@@ -11,18 +11,7 @@
     get_plans_dot,
 )
 from planners.drivers.landmark_driver import get_landmarks
-from planners.drivers.planner_driver_datatype import PlanDict, PlanningResult
-
-
-def as_dict(obj: object) -> Dict[str, Any]:
```

Comment:
Removed as_dict function in favor of Pydantic native model dump

Reply:
Great!